_data/merchants.yml

```diff
@@ -179,6 +179,8 @@
       url: https://moneroaddress.org/
     - name: Ownbit
       url: https://ownbit.io
+    - name: NOWNodes: Explorer & Node APIs
```

Comment:
```suggestion
    - name: NOWNodes - Explorer & Node APIs
```

Reply:
Hi! Added https://github.com/monero-project/monero-site/issues/1130

Reply:
Perfect. Now please add the changes i suggested. This PR is broken at the moment.